src/main/resources/data/computercraft/lua/rom/programs/poster.lua

```diff
@@ -1,94 +1,180 @@
-local startColor = term.getTextColor()
+local startTColor = term.getTextColor()
 
 local function printUsage()
   local programName = arg[0] or fs.getName(shell.getRunningProgram())
-  print("Usage: ")
-  print(programName .. " <file.2dj or url> [count]")
-  print(programName .. " stop")
+  
+  term.setTextColor(colors.lightGray)
+  term.write("Usage: ")
+
+  term.setTextColor(colors.white)
+  print(("%s <file or url> [count] [options?]"):format(programName))
+  
+  term.setTextColor(colors.gray)
+  print("Options for .2dja: [ <index> | <from> <to> ]")
+  
+  term.setTextColor(colors.white)
+  print(("Type '%s stop' to cancel the print task"):format(programName))
 end
 
 -- Find a printer
 local printer = peripheral.find("poster_printer")
 if not printer then
-  error("No printer found", 0)
+  error("No printer found.", 0)
```

Comment:
This is inconsistent with CraftOS error patterns: https://github.com/search?q=repo%3Acc-tweaked%2FCC-Tweaked+error%28%22+language%3ALua&type=code&l=Lua